### DISCUSSION TITLE
Margin geometry command has no effect?

### DISCUSSION THREAD

**Discussion by rjp23** [posquit0/Awesome-CV#494] [Q&A]
This is a great style and really helpful.

My only issue is that I can't seem to control the margins at all.

Edit: There's a geometry call in both the cv.tex and the style class itself. It ignores the one in the style class (probably as expected!).

**Comment by dcsim** [comment]
That is expected behavior. Glad you got it sorted.

@rjp23 or @OJFord: Either of you feel like closing this?
